test/neither.js

```diff
@@ -0,0 +1,62 @@
+import { Just, Nothing } from 'monet';
+import sinon from 'sinon';
+
+import * as RA from '../src/index';
+import eq from './shared/eq';
+
+
+const supportsFantasyLand = () => {
+  try {
+    return RA.neither(Just(false), Just(true)).equals(Just(false));
+  } catch (e) {
+    return false;
+  }
+};
+
+describe('neither', function() {
+  const isFantasyLandSupported = supportsFantasyLand();
+
+  it('combines two boolean-returning functions into one', function() {
+    const even = x => x % 2 === 0;
+    const gt10 = x => x > 10;
+    const f = RA.neither(even, gt10);
+
+    eq(f(8), false);
+    eq(f(13), false);
+    eq(f(14), false);
+    eq(f(9), true);
+  });
+
+  it('accepts functions that take multiple parameters', function() {
+    const between = (a, b, c) => a < b && b < c;
+    const total20 = (a, b, c) => a + b + c === 20;
+    const f = RA.neither(between, total20);
+
+    eq(f(4, 5, 11), false);
+    eq(f(12, 2, 6), false);
+    eq(f(5, 6, 15), false);
+    eq(f(12, 2, 7), true);
+  });
+
+  it('does not evaluate the second expression if the first one is true', function() {
```

Comment:
would already use a context/specify. I know you we trying to be consistent with ramda, but this make more sense from the perspective of viewer of the tests
```js
context('when the result of first function is true', function () {
   specify('should not evaluate the second expression', function () {
     ...
   });
));